apps/ledger-live-mobile/src/components/SendRowsFee.tsx

```diff
@@ -34,22 +34,24 @@ export default ({
   navigation,
   route,
   setTransaction,
+  status,
   ...props
 }: Props) => {
   const mainAccount = getMainAccount(account, parentAccount);
 
   // eslint-disable-next-line no-prototype-builtins
   if (perFamily.hasOwnProperty(mainAccount.currency.family)) {
```

Comment:
I know it was already like this but could you fix this eslint error, please?
```suggestion
  const hasFamilyProperty = Object.prototype.hasOwnProperty.call(
    perFamily,
    mainAccount.currency.family,
  );

  if (hasFamilyProperty) {
```